sources/packages/web/src/views/institution/locations/offerings/LocationOfferingView.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <full-page-container>
+    <template #header>
+      <header-navigator
+        title="Study period offerings"
+        :routeLocation="routeLocation"
+        subTitle="View Request"
+      >
+      </header-navigator>
+      <program-offering-detail-header
+        class="m-4"
+        :headerDetails="{
+          ...initialData,
+          status: initialData.offeringStatus,
+        }"
+      />
+    </template>
+    <offering-form :data="initialData"></offering-form>
+  </full-page-container>
```

Comment:
This page must be exact replica of Edit Offering page for the institutions excluding request for change buttons.